cmd/gitops/beta/run/cmd.go

```diff
@@ -128,6 +132,12 @@ gitops beta run ./deploy/overlays/dev --timeout 3m --port-forward namespace=dev,
 	cmdFlags.BoolVar(&flags.NoBootstrap, "no-bootstrap", false, "Disable bootstrapping at shutdown.")
 	cmdFlags.BoolVar(&flags.SkipResourceCleanup, "skip-resource-cleanup", false, "Skip resource cleanup. If not specified, the GitOps Run resources will be deleted by default.")
 
+	cmdFlags.StringVar(&flags.HiddenSessionName, "x-session-name", "", "The session name acknowledged by the sub-process. This is a hidden flag and should not be used.")
+
+	if err := cmdFlags.MarkHidden("x-session-name"); err != nil {
+		return nil
```

Comment:
❔ What is the reason for returning `nil` if marking this flag as hidden results in an error?

Reply:
good question actually.

what do you think would be used instead of returning nil here?